Analyzer errors in the travel_app, causing presubmit test warnings

The presubmit tests show these errors:

```
Run flutter analyze --fatal-infos
Resolving dependencies...
Downloading packages...
  characters 1.4.0 (1.4.1 available)
  material_color_utilities 0.11.1 (0.13.0 available)
Got dependencies!
2 packages have newer versions incompatible with dependency constraints.
Try `flutter pub outdated` for more information.
Analyzing travel_app...                                         

   info • 'appleProvider' is deprecated and shouldn't be used. Use providerApple instead. This parameter will be removed in a future major release • lib/main.dart:20:5 • deprecated_member_use
   info • 'androidProvider' is deprecated and shouldn't be used. Use providerAndroid instead. This parameter will be removed in a future major release • lib/main.dart:21:5 • deprecated_member_use

2 issues found. (ran in 12.6s)
Error: Process completed with exit code 1.
```

--- a/examples/travel_app/lib/main.dart
+++ b/examples/travel_app/lib/main.dart
@@ -17,8 +17,8 @@ void main() async {
   WidgetsFlutterBinding.ensureInitialized();
   await Firebase.initializeApp(options: DefaultFirebaseOptions.currentPlatform);
   await FirebaseAppCheck.instance.activate(
-    appleProvider: AppleProvider.debug,
-    androidProvider: AndroidProvider.debug,
+    providerApple: const AppleDebugProvider(),
+    providerAndroid: const AndroidDebugProvider(),
     providerWeb: ReCaptchaV3Provider('debug'),
   );
   await loadImagesJson();

